book/website/code_quality/code_quality.md

```diff
@@ -20,9 +23,28 @@ Tools for static code analysis can report bugs as well as style issues without e
 
 ## Static code analysis
 
-Static code analysis tools, also known as linters, can analyze source code and find bugs before your code is executed.
-In addition to finding bugs, many of these tools can also help maintain a consistent coding style.
-Examples of linter tools include [prospector](https://prospector.readthedocs.io) for Python, [lintr](https://github.com/jimhester/lintr) for R, or [shellcheck](https://www.shellcheck.net) for shell scripts.
+
+Static code analysis is a method that examines code and detects software vulnerabilities before before your code is executed or project is built and deployed. This analysis is capable of identifying quality issues, including security weaknesses and errors. In addition to finding bugs, many of these tools can also help maintain a consistent coding style.
```

Comment:
```suggestion
Static code analysis is a method that examines code and detects software vulnerabilities before your code is executed or the project is built and deployed. 
This analysis is capable of identifying quality issues, including security weaknesses and errors. 
In addition to finding bugs, many of these tools can also help maintain a consistent coding style.
```

Reply:
Line breaks for each new line for consistency (defined in the style guide of the contributing file)